Options.lua

```diff
@@ -701,11 +598,39 @@ function WarpDeplete:InitOptions()
         }),
 
         group(L["Objectives"], true, {
-          font(L["Objectives font"], "objectivesFont", "UpdateLayout"),
-          range(L["Objectives font size"], "objectivesFontSize", "UpdateLayout"),
-          fontFlags(L["Objectives font flags"], "objectivesFontFlags", "UpdateLayout"),
+          font(L["Objectives font"], "objectivesFont", "UpdateLayout", { width = 3 / 2 }),
+          fontFlags(L["Objectives font flags"], "objectivesFontFlags", "UpdateLayout", { width = 3 / 2 }),
+          range(L["Objectives font size"], "objectivesFontSize", "UpdateLayout", { width = 3 / 2 }),
+          {
+            type = "select",
+            name = L["Objectives time difference"],
+            desc = L["How to display timing differences in the objective display"],
+            sorting = {
+              "hidden",
+              "bestDiff",
+              "lastDiff"
+            },
+            values = {
+              ["hidden"] = L["Hidden"],
+              ["bestDiff"] = L["Difference to best kill time"],
+              ["lastDiff"] = L["Difference to last kill time"]
+            },
+            hidden = function() return not WarpDeplete.db.profile.timingsEnabled end,
+            get = function(info) return WarpDeplete.db.profile.timingsDisplayStyle end,
+            set = function(info, value)
+              WarpDeplete.db.profile.timingsDisplayStyle = value
+              self:UpdateLayout()
+            end,
+            width = 3 / 2
+          },
           color(L["Objectives color"], "objectivesColor", "UpdateLayout"),
           color(L["Completed objective color"], "completedObjectivesColor", "UpdateLayout"),
+          color(L["Improved time color"], "timingsImprovedTimeColor", "UpdateLayout", {
+            desc = L["The color to use when a boss kill time difference is less than or equal to zero"]
+          }),
+          color(L["Worse time color"], "timingsWorseTimeColor", "UpdateLayout", {
+            desc = L["The color to use when a boss kill time difference is greater than zero"]
+          }),
```

Comment:
```suggestion
          color(L["New best objective clear time"], "timingsImprovedTimeColor", "UpdateLayout", {
            desc = L["The color to use when you've set a new best objective clear time"]
          }),
          color(L["Slower objective clear time"], "timingsWorseTimeColor", "UpdateLayout", {
            desc = L["The color to use for objective clear times slower than your best time"]
          }),
```

Not 100% sure on these, but the previous ones are a little hard to understand tbh

Reply:
Never been good at naming stuff 😅